html/pfappserver/root/static.alt/src/views/Configuration/_config/captivePortal.js

```diff
@@ -16,6 +16,17 @@ export const view = (form = {}, meta = {}) => {
     {
       tab: null,
       rows: [
+        {
+          label: i18n.t('IP address'),
+          text: i18n.t('The IP address the portal uses in the registration and isolation networks. Do not change unless you know what you are doing.'),
```

Comment:
Maybe just a note that you need to restart haproxy-portal and keepalived.

Reply:
I added a note to restart all the services since that IP is in a lot of places and may appear in other places at some point